src/index.js

```diff
@@ -89,96 +97,103 @@ const usage = commandLineUsage([
+    {name: "quiet", alias: "q", type: Boolean, defaultValue: false},
+    {name: "no-box", alias: "Q", type: Boolean, defaultValue: false},
+    {name: "add-tld", alias: "t"}
+])
 
-if (args.version) {
+if (arguments_.version) {
     console.log("v2.0.0")
-    process.exit(0)
+    process.exit()
 }
 
-if (args.help) {
+if (arguments_.help) {
     // Show usages if has argument "--help"
     console.log(usage)
-    process.exit(0)
+    process.exit()
 }
 
-if (args.quiet && args.verbose) {
+if (arguments_.quiet && arguments_.verbose) {
     // quiet and verbose cannot be used at the same time
     console.warn(`${chalk.yellowBright.inverse.bold(`  ${figures.warning}  `)} ${chalk.bold("--quiet")} and ${chalk.bold("--verbose")} cannot be used at same time! Replaced with default value!`)
-    args.quiet = false
-    args.verbose = false
+    arguments_.quiet = false
+    arguments_.verbose = false
 }
 
 // Ping -> OK domains
 const aliveDomain = []
 
-// Check stdin (no input -> undefined)
-const stdin = (() => {
-    try {
-        return fs.readFileSync(process.stdin.fd, "utf-8")
-    } catch (e) {
-        return undefined
-    }
-})()
+// Check stdin (no input -> empty)
+const stdin = await getStdin()
 
-if (stdin !== undefined) 
-    args.domain = [ ...args.domain, ...stdin.trim().split(" ") ]
-// If stdin has an input, merge domains
+if (stdin)
+    arguments_.domain = [ ...arguments_.domain, ...stdin.trim().split(" ") ] // If stdin has an input, merge domains
 
 // If also not, show cursor and interactive prompt
-if (!("domain" in args)) {
+if (!("domain" in arguments_)) {
     cliCursor.show()
-    const tmp = await inquirer.prompt({
-        type: "input",
-        name: "domain",
-        message: "Which domain do you want to check (can split the spaces):"
-    }).then(ans => ans)
+
+    const domainAnswer = await Enquirer.prompt({
+        type: "list",
+        name: "domains",
+        message: "Which domain do you want to check (comma-separated)"
+    }).domains
+
+    arguments_.domain = [...domainAnswer]
+
     cliCursor.hide()
-    args.domain = [...tmp.domain.split(" ")]
 }
 
 const addTld = []
 
-if (args["add-tld"] === null) {
+if (arguments_["add-tld"] === null) {
     cliCursor.show()
-    const tmp = await inquirer.prompt({
-        type: "input",
+
+    const tldAnswer = await Enquirer.prompt({
+        type: "list",
         name: "tlds",
-        message: "Which top-level domain do you want to check (can split the spaces):"
-    }).then(ans => ans)
+        message: "Which top-level domain do you want to check (comma-separated):"
+    }).tlds
+
+    addTld.push(...tldAnswer)
+
     cliCursor.hide()
-    addTld.push(...tmp.tlds.split(" "))
 }
 
 // Ping
-const gets = domains => {
-    domains.forEach(domain => {
-        ping.promise.probe(domain, undefined)
-            .then(res => {
-                if (res.alive) {
+const gets = (domains) => {
+    domains.forEach((domain) => {
+        ping.promise.probe(domain)
```

Comment:
Codacy found an issue: [Arrow function has a complexity of 6. Maximum allowed is 4.](https://app.codacy.com/gh/P2P-Develop/TopDomainChecker/pullRequest?prid=6274849)